### ISSUE TITLE
[semantic-issue] Add semantics for sklearn train_test_split

### ISSUE THREAD

**Issue by osryde** [spangea/Pyra#180]
**Description**
Divides a dataset into two sets: one for training a machine learning model and the other for testing or evaluating the model. 

**Minimal running example**
```
x_train, x_test, y_train, y_test = train_test_split(Xdata_scaled, y, test_size=0.25, random_state=5)

```

**Links to documentation**
https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html

**Note (optional)**

Notebook: 0-91-accuracy-score-for-4-models-lr-rc-gbc-rfc.py
